client/elements/menus/sc-language-menu.html

```diff
@@ -184,6 +189,18 @@
               this.loadResources(this.resolveUrl(`/localization/elements/sc-language-menu/${this.language}.json`));
           }
 
+          _computeLanguageMenu() {
+            const languageMenu = [];
+            for (let item in this.languageChoice) {
+              if (this.languageChoice[item].lang in this.rootLangMappings) {
```

Comment:
it's better if we use the `languageMenu` that was passed by `computed` as an argument into this function and operate on that instead of referring to `this.languageMenu`. If we pass `rootLangMappings` in there too, it becomes a [pure function](https://en.wikipedia.org/wiki/Pure_function), which is very good.